business_objects/embedding.py

```diff
@@ -208,6 +211,14 @@ def create(
     if state:
         embedding.state = state
 
+    if started_at:
+        embedding.started_at = started_at
+
+    if finished_at:
+        embedding.finished_at = finished_at
+    else:
+        embedding.finished_at = sql.func.now()
```

Comment:
im not sure this is right? when we create an embedding and don't provide a finished at its set to now?